tests/Jellyfin.Server.Implementations.Tests/IO/ManagedFileSystemTests.cs

```diff
@@ -26,7 +28,16 @@ public ManagedFileSystemTests()
             string expectedAbsolutePath)
         {
             var generatedPath = _sut.MakeAbsolutePath(folderPath, filePath);
-            Assert.Equal(expectedAbsolutePath, generatedPath);
+
+            if (MediaBrowser.Common.System.OperatingSystem.Id == OperatingSystemId.Windows)
+            {
+                var expectedWindowsPath = "d:" + expectedAbsolutePath.Replace('/', '\\');
```

Comment:
Will this always be the D drive?
Why don’t you just strip everything before the ‘:’?